sqlserver/datadog_checks/sqlserver/activity.py

```diff
@@ -146,26 +157,60 @@ def _get_activity(self, cursor):
     def _normalize_queries_and_filter_rows(self, rows, max_bytes_limit):
         normalized_rows = []
         estimated_size = 0
+        rows.sort(key=lambda r: self._get_sort_key(r))
```

Comment:
Avoid side effects; this will mutate the rows passed in. Should be:
```suggestion
        rows = sorted(rows, key=lambda r: self._get_sort_key(r))
```